powerstore/datasource_filesystem_snapshot.go

```diff
@@ -162,6 +163,14 @@ func (d *fileSystemSnapshotDataSource) Read(ctx context.Context, req datasource.
 		}
 	}
 
+	if len(fileSystemSnapshots) == 0 {
+		resp.Diagnostics.AddError(
+			"Unable to find PowerStore File Systems Snapshots",
```

Comment:
As confirmed from Akash, If it is returning empty array then that is fine. we don't need to show any error. It can remain as empty list

Reply:
As per team recommendation, I will be closing this PR.